src/components/Card.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const isInitialCard = (card) => {
+  return Object.values(card).some((value) => value !== '');
+};
+
+function Card({ card }) {
+  const [isEmptyCard, setIsEmptyCard] = useState(true);
+
+  useEffect(() => {
+    return isInitialCard(card) ? setIsEmptyCard(false) : setIsEmptyCard(true);
+  }, [card]);
```

Comment:
클린업에서 해당 로직이 수행되도록 한 게 의도된 동작인가용? 어떤 이유에서 클린업 함수에서 동작해야하는 지 궁금해요!
```suggestion
  useEffect(() => {
    isInitialCard(card) ? setIsEmptyCard(false) : setIsEmptyCard(true);
  }, [card]);
```

Reply:
제가 잘 못 작성하였습니다. ㅠ
아래 코멘트를 보고 수정하였습니다.